.clabot

```diff
@@ -1,5 +1,5 @@
 {
-  "message": "Thank you for your pull request and welcome to our community. We require contributors to sign our [Contributor License Agreement](https://github.com/retest/clabot-config/blob/master/Contributor-License-Agreement.md), and we don\"t seem to have the users {{usersWithoutCLA}} on file. In order for us to review and merge your code, for each noted user ***please add yourself to the [.clabot](https://github.com/retest/clabot-config/edit/master/.clabot) file***.",
+  "message": "Thank you for your pull request and welcome to our community. We require contributors to sign our [contributor license agreement (CLA)](https://github.com/retest/clabot-config/blob/master/Contributor-License-Agreement.md), and we don\"t seem to have the user(s) {{usersWithoutCLA}} on file. In order for us to review and merge your code, for each noted user ***please add the GitHub username to the [.clabot](https://github.com/retest/clabot-config/edit/master/.clabot) file***.",
```

Comment:
`add **your** GitHub username` 
I prefer it when each person does it for themselves.